client/asset/dcr/dcr.go

```diff
@@ -3005,6 +3018,296 @@ func (dcr *ExchangeWallet) EstimateRegistrationTxFee(feeRate uint64) uint64 {
 	return (dexdcr.MsgTxOverhead + dexdcr.P2PKHOutputSize*2 + inputCount*dexdcr.P2PKHInputSize) * feeRate
 }
 
+// MakeBondTx creates a time-locked fidelity bond transaction. The V0
+// transaction has two required outputs:
+//
+// Output 0 is a the time-locked bond output of type P2SH with the provided
+// value. The redeem script looks similar to the refund path of an atomic swap
+// script:
+//
+//	<locktime[4]> OP_CHECKLOCKTIMEVERIFY OP_DROP <pubkey[33]> OP_CHECKSIG
+//
+// The pubkey referenced by the script comes from the wallet, but in general, it
+// need not be controlled by the wallet.
+//
+// Output 1 is a DEX Account commitment. This is an OP_RETURN output that
+// references the provided account ID.
+//
```

Comment:
In the PR description you mentioned that maybe there are other services with similar bond transactions. Do you know of any others? One way to solve this would be to put another number that all DEX registration bonds have to have into this `OP_RETURN` output.